src/controllers/createpdf.controller.ts

```diff
@@ -0,0 +1,92 @@
+import { Controller, Get, Res, Post, Body, Render, Param, Query} from '@nestjs/common';
+import {Response} from 'express';
+import {createUserData, createLogInfo,createRunConf} from '../dtos/createpdf.dto';
+import {CreatePdfService} from "../services/createpdf.service";
+import { Log } from '../entities/log.entity';
+import { getManager } from 'typeorm';
+import { Run } from '../entities/run.entity';
+import { ApiUseTags, ApiBearerAuth } from '@nestjs/swagger';
+
+@ApiUseTags('createpdf')
+@Controller('createpdf')
+export class CreatepdfController {
+  constructor(public createPdfService: CreatePdfService){};
+
+  @Get('getlog')
+  async getLogInfo(@Query() query: createLogInfo, @Res() res: Response): Promise<void>{
+    var allDate = this.createPdfService.getDate();
+
+    const repository = getManager().getRepository(Log);
+    
+    const result = await repository.createQueryBuilder("log")
+      .leftJoinAndSelect("log.user", "user_id, external_id, sams_id")
+      .where("log.log_id = :id", {id: query.id})
+      .getOne();
+
+    let params = {user:result["user"]["userId"], external:result["user"]["externalUserId"], sams:result["user"]["samsId"], logId:result["logId"],
+    subsytemID:result["subsystemFkSubsystemId"], subtype:result["subtype"], origin:result["origin"], creationTime:result["creationTime"],
+    title:result["title"], text:result["text"]};
+    return this.createPdfService.createPdf(params, `Log ${allDate}.pdf`, "/templates/LogInfoTemplate.ejs", res);   
```

Comment:
Output file name is confusing/non identifying.
It creates a the following name `Log HH_MM DD.MM.YYY.pdf.pdf`. This is wrong for the following reasons:
- Downloading multiple files within the same time frame will overwrite the already downloaded file.
- Two .pdf extensions when downloading the file.
- The character `:` is not allowed in file names, it will show up as _.
- Please do not use spaces in file names.

Suggestions:
- Use epoch time instead of whatever monstrosity you are currently using to get the date. This ensures that the downloaded file will always be unique.
- Create a file with following name/template `Log_{logId}_{epoch-time}.pdf`